### DISCUSSION TITLE
No way to configure route53 to use HttpApi from @aws-cdk/aws-apigatewayv2-alpha?

### DISCUSSION THREAD

**Discussion by azhong-git** [aws/aws-cdk#22448] [Q&A]
Hi,

I am trying to add a custom domain to an `HttpApi` however was stuck at assigning `ARecord` using `route53`. However `route53_targets` does not support `HttpApi` in alpha ([doc](https://docs.aws.amazon.com/cdk/api/v2/docs/aws-cdk-lib.aws_route53_targets-readme.html)). 

Any help is appreciated. 

(In my usecase I would like to use HTTP API instead of REST API)

**Comment by sladg** [comment]
Facing same issue, any way how to deal with this?

**Reply by azhong-git** [reply]
Hi @sladg, op here. I ended up using `RecordTarget.fromAlias(new ApiGatewayv2DomainProperties...` to setup the route53 target with `HttpApi`. (Sorry I forget to update this question)
```
    const domain = new DomainName(this, "domain-name", {
        domainName: domainName,
        certificate: certificate,
        securityPolicy: securityPolicy,
      });
      httpApi = new HttpApi(this, "http-api", {
        description: description,
        defaultDomainMapping: {
          domainName: domain,
        },
      });
      new ARecord(this, "example-a-record", {
        recordName: domainName,
        zone: hostedZone,
        target: RecordTarget.fromAlias(
          new ApiGatewayv2DomainProperties(
            domain.regionalDomainName,
            domain.regionalHostedZoneId
          )
        ),
      });
```

**Reply by sladg** [reply]
Nice, that looks a bit cleaner for sure! Thanks!
